### ISSUE TITLE
Dice System

### ISSUE THREAD

**Issue by jakesower** [jakesower/rpg-2024#9]
Perhaps my favorite part of this new system is its dice pool mechanic.

To recap:

- Should a player have fewer than 2 dice, d10s are added until the player has 2 dice.
- Roll all the dice at once.
- Should all the dice show the same number, roll an additional 1d10. Keep doing this until a repeat is not rolled.
- Remove all dice showing "0".
- Group the remaining dice to make doubles, triples, etc.
- The success of the action depends on these groupings.

This is a long winded and precise way of saying "grab a fistful of d6s and d10s, roll them and count tuples."

Here are some of the qualities that I like about it:

1. d6s are much better than d10s. Have d6s be primarily (exclusively?) tied to skills really rewards leveling up skills.
1. Since the number of d6s in a pool never exceeds 5, and "0" rolls always fail, it's always possible to fail any given check
1. Since you can keep adding d10s if you roll all the same thing, big successes are always possible
1. +1 bonuses hit a sweet spot of being good without being too powerful. They roughly map to somewhere between a +1 and a +4 in D&D, with the bigger bonuses being on the lower probability end of the spectrum. (I encourage you to look at the probability tables.)
1. The real winner though is how you can tie things to various tuples. Fireballs can get bigger on triples, war hammers can delay people on quadruples, etc. Varying on odd triples and even triples adds even more room to tinker with things. The big advantage of this is that you can include mechanics like disarm, metamagic, and others without feats or abilities. It lets you make weapons feel and act differently without making them unbalanced.
1. You can play with the dice pool itself for different effects. The "Lucky 7" spell adds a "7" to every dice pool. While not helping skilled d6s, it can benefit characters using d10-heavy rolls.
1. Triples "come online" once dice pools reach a certain size. This allows abilities to be crafted that don't do anything on doubles, but do on triples. These would be represented as higher level spells or advanced feats otherwise.
1. Attacks can be varied with the mechanic. Big dice pools with weak tuple effects for reliable attacks; small dice pools with strong triple/quadruple effects for more randomly inclined attacks. Think quarterstaff vs crossbow.

**Comment by jakesower** [comment]
@matt439miller said

> It's a mixed dicepool, which is weird. Makes it hard to estimate chance of success, both players and dm. Big tables don't really lend Themselves to memory,or even a heuristic of how many dice is how many tuples. I'm not sure the 'ignore zeroes' is helpful.

**Comment by matt439miller** [comment]
"Should all the dice show the same number, roll an additional 1d10. Keep doing this until a repeat is not rolled." - what is the point of this?

**Comment by jakesower** [comment]
> "Should all the dice show the same number, roll an additional 1d10. Keep doing this until a repeat is not rolled." - what is the point of this?

Small chance for more successes than would otherwise be possible. That and the "don't count 0s" are there for the effect of every roll being capable of success and failure.

**Comment by matt439miller** [comment]
"0" in the sense of having zero tulles (successes), ok.

**Comment by jakesower** [comment]
"0s" or "10s" whichever you like for a d10. The point is that rolling them is always bad.

**Comment by jakesower** [comment]
> Makes it hard to estimate chance of success, both players and dm. Big tables don't really lend Themselves to memory,or even a heuristic of how many dice is how many tuples.

The tables of probabilities would not be made available to players. The idea is Skill/Attribute +/- Circumstances for rolls. (Or in some cases Skill/Skill +/- Circumstances). That should hopefully be easy enough for people to grok as they play. The subtleties of how the probabilities are affected is only useful in design.

5e has a little bit of this. Is it better to get a +5 to a check, or advantage? It depends on the DC of the check. Advantage is one of those mechanics that's hard to pin down as far as probabilities go, but is clearly good. A +1 to a check in this game is always good in the same sense and spirit.

**Comment by matt439miller** [comment]
> @matt439miller said
> 
> 
> 
> > It's a mixed dicepool, which is weird. Makes it hard to estimate chance of success, both players and dm. Big tables don't really lend Themselves to memory,or even a heuristic of how many dice is how many tuples. I'm not sure the 'ignore zeroes' is helpful.

My intuition is that 2d10 is going to succeed ten percent of the time, so there is always a chance. And 2d6 is going to succeed maybe 1 in 6, or almost never. Whereas, 5d6 should net me... 2.5 in 6 chances, or worse than even? With each d10 on that adding a less 1/6 chance. And with a 1d6 and 1d10, there are...6 tuples? So a 6 in 60 chance. And with three dice... I have ducking idea.

**Comment by jakesower** [comment]
> It's a mixed dicepool,

Yes, this is one of those tricky aspects I'm on the fence about. For the way it affects the numbers, it's really fucking good. d6s are quite a lot better than d10s and really reflect the value of skills. WoD is very attribute based and I can't say I particularly like it. 5e is the same when it comes down to it.

That said, having two types of dice does increase complexity. I tried something with more than 2 types of dice and it was awful. Keeping d6s always connected to a skill makes it more manageable since the roller is always dealing with the number of d6s and everything else is just d10s.

**Comment by jakesower** [comment]
> > @matt439miller said
> > > It's a mixed dicepool, which is weird. Makes it hard to estimate chance of success, both players and dm. Big tables don't really lend Themselves to memory,or even a heuristic of how many dice is how many tuples. I'm not sure the 'ignore zeroes' is helpful.
> 
> My intuition is that 2d10 is going to succeed ten percent of the time, so there is always a chance. And 2d6 is going to succeed maybe 1 in 6, or almost never. Whereas, 5d6 should net me... 2.5 in 6 chances, or worse than even? With each d10 on that adding a less 1/6 chance. And with a 1d6 and 1d10, there are...6 tuples? So a 6 in 60 chance. And with three dice... I have ducking idea.

My point is that worrying about the math during play isn't helpful. If I have a DC 15 check in 5e, I (personally) am not going to be able to calculate how much advantage would improve my odds of succeeding. I just know that it would indeed help me and roll with it.

**Comment by matt439miller** [comment]
See, the fact that advantage adds 3.x to the roll is anchored in my brain.

**Comment by jakesower** [comment]
The obscure math of it does have its advantages and payoffs as well. Tying abilities to tuples is a hell of a lot of fun. For instance, fireballs get bigger on triples. Simple to understand, fun to play. The fact that you might not be able to calculate that probability in your head is fine. You "3.x to the roll" is a useful mnemonic that can have analogues in any system where they're useful.

(Also, there are rules of thumb for the people who care to think of the probability table. +1 is about +10% for triples, more for doubles, less for quadruples.)

**Comment by matt439miller** [comment]
So, was just watching a Matt Coleville video where he talks about upping the ac of a baddy by +1, and he point out that might matter maybe one in the 20 attacks 4x players will make over 5 rounds. Got me thinking. If you are a player, and you don't know the TN you need to hit a guy, but you know a roll of x did or did not. So you roll a 12, that misses, and it's a bit of an oh shit moment, because it means you've got a 40 percent chance to hit them. It also means their bonus to hit you is going to be big, and you have a fierce enemy on your hands and will need help to defeat. But if you hit someone on an 8, you know that guy is just meat.

**Comment by matt439miller** [comment]
For melee characters, the whole game is making calculations, and seeking conditions (or powers) that allow you to adjust what you do when your 12 misses. It's what makes battlemaster so much better than champion to play in 5e.  Because once in melee, bugging out not an option, unless you are a skirmisher. The good vibe is when you do everything possible to make the fight favor you before melee starts. Maybe you get lucky with a thrown weapon. Maybe you get the first strike in. Maybe you disarm the other guy, or knock him down, or whatever. But what you get back, in terms of effects of a roll, is very important.

**Comment by jakesower** [comment]
8 vs 12 isn't a huge gap, really. It does _feel_ that way. It feels like a 50% difference, but it's only 25% (which is certainly nontrivial). There are a lot of ways of resolving that situation.

- Try to modify circumstances to get those sweet, sweet bonuses and advantage
- Use a weapon that's more reliable on getting the hits, even if they're not as impactful when they hit
- Go the opposite way and try to get that home run hit

I don't know that I agree with the calculation mindset. d20 is inherently wildly variable. Roll a 2 and you can count on being fucked. An 18 is almost always going to hit. Trying to do math against something so heavily random has limited payoff. Sure, the law of large numbers will make something of it, but any given battle hinges as much (if not more) on the dice as the tactics.

I feel like 5e is very much about getting advantage and not disadvantage. "Aid other" and those sorts of abilities that deal with advantage make a big difference, but are really dull.

I do like having the first striking thrown weapon.

The low risk/regularish damage vs high risk/big damage + cool effects split is more interesting. I'd like something that promotes those kinds of choices. It's why order/chaos magic is such an interesting split for me.

**Comment by matt439miller** [comment]
I think having the oppty to modify is key. Nobody wants to feel boned when they roll they expect to hit with misses.

**Comment by matt439miller** [comment]
> "0s" or "10s" whichever you like for a d10. The point is that rolling them is always bad.

Ah, ok, because a ten can roll a donut but a six can't, so if you are relying on tens, there is a chance of a dud dice, like rolling a 1.

**Comment by matt439miller** [comment]
> > It's a mixed dicepool,
> 
> 
> 
> Yes, this is one of those tricky aspects I'm on the fence about. For the way it affects the numbers, it's really fucking good. d6s are quite a lot better than d10s and really reflect the value of skills. WoD is very attribute based and I can't say I particularly like it. 5e is the same when it comes down to it.
> 
> 
> 
> That said, having two types of dice does increase complexity. I tried something with more than 2 types of dice and it was awful. Keeping d6s always connected to a skill makes it more manageable since the roller is always dealing with the number of d6s and everything else is just d10s.

In a way, that's really nice, because it provides a check off the mix of your pool. You'll mostly be rolling d6 and trying to sneak the odd d10 in there.  But... pool caps at five so what if I have 5d6?

**Comment by matt439miller** [comment]
> "0s" or "10s" whichever you like for a d10. The point is that rolling them is always bad.

I continue to be concerned, and wonder if the exploding dice is worth the additional mechanics on what's already pretty complex.

**Comment by jakesower** [comment]
Yeah, I'd be fine nixing the 0s fail rule if it's that bothersome.

Dice pools do not cap out at 5 dice. It caps out at 5d6, but any number of d10s are allowed. I'd imagine dice pools will be 8-10 dice for the most part. More dice for the super slow terror axe and other big stuff.

**Comment by matt439miller** [comment]
Ah, so it's the d6's that cap, and they cap because of... ability score? Hmm. 8-10 dice, but non-tuples don't matter, so you can chuck those from the get-go. Must be certain to not let the core rules including 'splitting' a quad... but that will be rare enough that it wont matter. I feel like I need to roll some dice, just to get a vibe on this. <did some excel>. Hassle to do in Excel, probably easier to do with making sets of dice (I used 5d6+5d10).  Two pairs and a thruple. Can I use all of those? Or can I just use one set to power my spell?

**Comment by matt439miller** [comment]
If you have less than two dice in your pool, add a d10.
Roll all the dice.
Group the dice to make doubles, triples, etc.
Sets of tens don't count.

Exploding rule not worth the hassle. Removing tens to end of operations makes it a contingent rule. Still not clear why it matters, thematically. Changes failure when roll 2d10 from being 90% to being even more certain? (Not worth it). So what if every roll, you get a tuple? That's like rolling over the DC, and the triples are the 'criticals'. If you're going to have a critial fail roll )questionable value) something weird like "double aughts kills the whole roll" seems better.

**Comment by jakesower** [comment]
You can use all the tuples you roll. You get all of the effects together at once. So your fireball could get 1 bigger AND cause burning.

I'm fine nixing the 0's don't count rule.

Also, I'd consider making rolls of 1-4 "successes" that contribute to the magnitude of success, with tuples throwing in flavor depending on what you're doing.
